sample/android-sdk-sample/app/src/main/java/com/bitmark/androidsdksample/fragments/QueryFragment.java

```diff
@@ -0,0 +1,130 @@
+package com.bitmark.androidsdksample.fragments;
+
+import android.app.Fragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.ProgressBar;
+import android.widget.Toast;
+
+import com.bitmark.androidsdksample.R;
+import com.bitmark.androidsdksample.utils.Global;
+import com.bitmark.apiservice.params.query.BitmarkQueryBuilder;
+import com.bitmark.apiservice.response.GetBitmarksResponse;
+import com.bitmark.apiservice.utils.record.BitmarkRecord;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.reactivex.Single;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+
+import static com.bitmark.androidsdksample.samples.QuerySample.queryBitmarks;
+import static com.bitmark.androidsdksample.utils.CommonUtil.setClipboard;
+
+public class QueryFragment extends Fragment {
+    CompositeDisposable compositeDisposable = new CompositeDisposable();
+    View view;
+    EditText accountNumberInputText;
```

Comment:
The prefixes should be
- `TextView` -> `tv` or `text`
- `EditText` -> `edt` or `edit`
- `ListView` -> `lv` 
- `RecyclerView` -> `rc`
- `Checkbox` -> `cb`